frontend/src/main/java/puf/frisbee/frontend/viewmodel/StartViewModel.java

```diff
@@ -48,19 +96,27 @@ public BooleanProperty getShowLoginRegisterButtonProperty() {
     }
 
     public BooleanProperty getShowStartButtonProperty() {
-        this.showStartButton.setValue(this.playerModel.isLoggedIn());
+        if (this.levelModel.getCurrentLevel() <= this.levelModel.getMaximumLevel()) {
```

Comment:
Alternative: `this.showStartButton.setValue(this.playerModel.isLoggedIn() && this.levelModel.getCurrentLevel() <= this.levelModel.getMaximumLevel());` 
Oder etwas lesbarer:
```
boolean showStartViewButton = this.playerModel.isLoggedIn() && this.levelModel.getCurrentLevel() <= this.levelModel.getMaximumLevel();
this.showStartButton.setValue(showStartViewButton);
```

Reply:
Ist gemacht.

Reply:
Erledigt.